errors/default/page.phtml

```diff
@@ -42,7 +42,7 @@
         <div class="page">
             <div class="header-container">
                 <div class="header">
-                    <a href="<?php echo $this->getBaseUrl()?>" title="Magento Commerce" class="logo"><img src="images/logo.gif" alt="Magento Commerce" /></a>
+                    <a href="<?php echo $this->getBaseUrl()?>" title="OpenMage" class="logo"><img src="images/logo.svg" alt="OpenMage" height=35 /></a>
```

Comment:
" are missing for height attribute :)

Reply:
done, although I think this note is completely useless, quotes should be added when necessary, most of all if you're so worried about removing "x=0" from the svg, then we can reduce the filesize removing unnecesary quotes.